src/stylesheets/components/_accordions.scss

```diff
@@ -4,70 +4,99 @@ $accordion-border: 3px solid $color-gray-lightest;
 
 // Accordion Styles
 
+@mixin accordion-list-styles {
+  @include unstyled-list;
+  color: $color-base;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+@mixin accordion-list-item-styles {
+  background-color: $color-gray-lightest;
+  font-family: $font-sans;
+  list-style: none;
+  margin-bottom: 6px;
+  width: 100%;
+}
+
+// scss-lint:disable PropertyCount
+@mixin accordion-button-styles {
+  @include button-unstyled;
+  background-color: $color-gray-lightest;
+  background-image: url('../img/minus.png');
+  background-image: url('../img/minus.svg');
+  background-position: right 3rem center;
+  background-repeat: no-repeat;
+  background-size: 13px;
+  color: $color-base;
+  cursor: pointer;
+  display: inline-block;
+  font-family: $font-sans;
+  margin: 0;
+  padding: 1.5rem 5.5rem 1.5rem 3rem;
+  width: 100%;
+
+  &:focus {
+    box-shadow: $focus-shadow;
+  }
+
+  &:hover {
+    background-color: $color-gray-lighter;
+    color: $color-base;
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    margin: 0;
+  }
+}
+// scss-lint:enable PropertyCount
+
+@mixin accordion-button-unopened-styles {
+  background-image: url('../img/plus.png');
+  background-image: url('../img/plus.svg');
+  background-repeat: no-repeat;
+  background-size: 1.3rem;
+}
+
 .usa-accordion,
 .usa-accordion-bordered {
+  @include accordion-list-styles;
+
   + .usa-accordion,
   + .usa-accordion-bordered {
     margin-top: 1rem;
   }
 
+  // TODO deprecated ruleset based on old accordion.
```

Comment:
Dis-regard, as this is reference to the upgrades vs. newer class based specificity.